ci-operator/config/openshift/kueue-operator/openshift-kueue-operator-release-1.0.yaml

```diff
@@ -178,6 +192,63 @@ tests:
           memory: 500Mi
     test:
     - chain: kueue-operator-test-e2e
+- always_run: false
+  as: test-e2e-disconnected
+  minimum_interval: 96h
+  steps:
+    cluster_profile: gcp-qe
+    env:
+      CATSRC_IMG: registry.redhat.io/redhat/redhat-operator-index:v4.19
+      CATSRC_NAME: redhat-operators
+      MIRROR_OPERATORS: "true"
+    post:
+    - as: kueue-must-gather-disconnected
+      cli: latest
+      commands: |
+        source "${SHARED_DIR}/env"
```

Comment:
Looks like this step is failing with:
`/bin/bash: line 3: /tmp/secret/env: No such file or directory`

Reply:
I think it works it just that the latest run had some infra issues.
https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/openshift_release/66357/rehearse-66357-periodic-ci-openshift-kueue-operator-release-1.0-test-e2e-disconnected/1943611025743941632/artifacts/test-e2e-disconnected/kueue-must-gather-disconnected/build-log.txt